app/views/shared/_profile_card_content.html.erb

```diff
@@ -57,26 +39,29 @@
           </div>
         </li>
       <% end %>
-      <% if actor.education.present? %>
-        <li>
-          <div class="key">
-            Education
-          </div>
-          <div class="value">
-            <%= actor.education %>
-          </div>
-        </li>
+      <% if (header_fields = actor.decorate.ui_attributes_for(area: :header)).present? %>
```

Comment:
We are now automatically adding all header fields (maximum 3) to the cards. /cc @aitchiss